src/Chart/Chart.jsx

```diff
@@ -45,8 +49,10 @@ export default (ChartElement) => class ChartComponent extends Component {
       data: dataset,
       datasets,
       xAxis,
-      yAxis
+      yAxis,
+      ySeriesField
     } = data
+    const ySeriesMap = new Map()
```

Comment:
I take back the reasoning here. we should be using babel-polyfill if we have those issues.

however, the question still stands.